src/d3a/models/strategy/pv.py

```diff
@@ -62,15 +62,25 @@ def __init__(
         self.state = PVState()
 
     @staticmethod
-    def _validate_constructor_arguments(panel_count, risk, max_panel_output_W,
-                                        initial_selling_rate):
-        if not (0 <= risk <= 100 and panel_count >= 1):
+    def _validate_constructor_arguments(panel_count=None, risk=None,
+                                        max_panel_output_W=None, initial_selling_rate=None):
+        if not ((risk is None or 0 <= risk <= 100) and
+                (panel_count is None or panel_count >= 1)):
             raise ValueError("Risk is a percentage value, should be "
                              "between 0 and 100, panel_count should be positive.")
-        if max_panel_output_W < 0:
+        if max_panel_output_W is not None and max_panel_output_W < 0:
             raise ValueError("Max panel output in Watts should always be positive.")
-        if initial_selling_rate < 0:
-            raise ValueError("Initial selling rate must be greater equal 0.")
+        if initial_selling_rate is not None and initial_selling_rate < 0:
+            raise ValueError("Min selling rate should be positive.")
+
+    def area_reconfigure_event(self, **kw):
```

Comment:
Very minor, but IMO kw is not self-explaining. What does it stand for? Keyword?

Reply:
Stands for kwargs == 'keyword arguments'. Since it is confusing as is, I will rename it to kwargs